gp2gp-translator/src/nonFunctionalTest/TestPlans/PS_NF_REL_1/PS_NF_REL_1.jmx

```diff
@@ -0,0 +1,781 @@
+                  <stringProp name="Scope.variable">statusCode</stringProp>
+                  <stringProp name="TestPlan.comments">Assert tha the response is either 204 (waiting) or 200 (success)</stringProp>
+                </ResponseAssertion>
+                <hashTree/>
+              </hashTree>
+              <TestAction guiclass="TestActionGui" testclass="TestAction" testname="Wait for 1 Second" enabled="true">
+                <intProp name="ActionProcessor.action">1</intProp>
+                <intProp name="ActionProcessor.target">0</intProp>
+                <stringProp name="ActionProcessor.duration">1000</stringProp>
+                <stringProp name="TestPlan.comments">Wait between requests </stringProp>
+              </TestAction>
+              <hashTree>
+                <UniformRandomTimer guiclass="UniformRandomTimerGui" testclass="UniformRandomTimer" testname="Pause" enabled="true">
+                  <stringProp name="ConstantTimer.delay">1000</stringProp>
+                  <stringProp name="RandomTimer.range">100</stringProp>
+                </UniformRandomTimer>
+                <hashTree/>
+              </hashTree>
+            </hashTree>
+          </hashTree>
+          <TransactionController guiclass="TransactionControllerGui" testclass="TransactionController" testname="${threadDetails} 1: Switch - RCMR_IN030000UK06 - Invalid" enabled="true">
+            <boolProp name="TransactionController.includeTimers">false</boolProp>
+            <boolProp name="TransactionController.parent">true</boolProp>
+            <stringProp name="TestPlan.comments">A transaction controller to send  an RCMR_IN030000UK06 message flow with an invalid message</stringProp>
+          </TransactionController>
+          <hashTree>
+            <PublisherSampler guiclass="JMSPublisherGui" testclass="PublisherSampler" testname="${threadDetails} Send  Invalid RCMR_IN30000UK06 message" enabled="true">
+              <stringProp name="jms.jndi_properties">false</stringProp>
+              <stringProp name="jms.initial_context_factory">org.apache.qpid.jms.jndi.JmsInitialContextFactory</stringProp>
+              <stringProp name="jms.provider_url">${__P(jndiPropertiesPath, ../../Configuration/jndi.properties)}</stringProp>
+              <stringProp name="jms.connection_factory">qpidConnectionFactory</stringProp>
+              <stringProp name="jms.topic">mhsQueueName</stringProp>
+              <stringProp name="jms.expiration"></stringProp>
+              <stringProp name="jms_error_reconnect_on_codes"></stringProp>
+              <stringProp name="jms.priority"></stringProp>
+              <stringProp name="jms.security_principle"></stringProp>
+              <stringProp name="jms.security_credentials"></stringProp>
+              <stringProp name="jms.text_message"></stringProp>
+              <stringProp name="jms.input_file">/Users/martinwheeler/Documents/InvalidMessages/RCMR_IN030000UK06.txt</stringProp>
```

Comment:
Would these properties need to be changed if someone else ran the tests? If so, should they be moved to config.properties?

Reply:
These were already in config.properties but this was not reflected in the jmx file.  Updated all occurrences of local references